database/entity/0046-messages_tables/Message.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseEntity, Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm'
+import { Contribution } from '../Contribution'
+
+@Entity('messages', { engine: 'InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci' })
```

Comment:
warum nur "Message" statt "ContributionMessage"?
Ja es sind die ersten Messages, die wir in die Anwendung einbauen, aber sobald weitere Messages mit anderem fachlichen Kontext dazu kommen bedarf es hierfür Refactoring-Aufwände, die vermeidbar wären